### DISCUSSION TITLE
Totaly desparate

### DISCUSSION THREAD

**Discussion by Ragrati** [NightscoutFoundation/xDrip#3577] [Q&A]
Hi, 
I would like to find a connected watch that can allow me to check my blood sugar levels. I ordered around ten, but I have a Huawei P30 and therefore none fit BUT!
I saw that you could connect Mi Band so I bought a Xiaomi Mi Band Pro 8 but I can't do it!
Maybe watchrip doesn't go that far? Is it possible ? Should I change to an older one?
I don't want to change my smartphone and also switch to wear OS: long battery life is good (even if I would like to have Google maps)

So: is it possible to send from juggluco to xdrip on my smartphone and then on the xiaomi?
Thank you for helping me, I'm going to go hiking and it would be really good to have the blood sugar levels I already have by juggluco

**Comment by PhilTU** [comment]
Watchdrip.org will help with compatibility.
Juggluco can send to watchdrip direct.

Juggluco on WearOS watch doesn't need a phone.

**Comment by GLoren1** [comment]
I've been trying to get my Mi Band 7 to work with Watchdrip for Dexcom, and it's not reliable.  At least I haven't been able to get it to work correctly.
